README.md

```diff
@@ -94,10 +95,10 @@ suite.run()
 #### Using the options map on the static `test` method:
 
 ```js
-import "./testie" for Testie
-import "./src/tap_reporter" for TapReporter
+import "wren-testie/testie" for Testie
+import "wren-testie/reporters" for TapReporter
```

Comment:
Perhaps a `reporters` directory?
```js
import "wren-testie/reporters/tap" for TapReporter
import "wren-testie/reporters/cute" for CuteReporter
```

Reply:
Ah, this would be even better if you want to refactor it that way... and then just pull in the code one file at a time without a top level `reporters`

Reply:
And actually an "index" script that imports stuff is bad: TapReporter sets the "__withColors" _class variable_ for Expect and StackTraceReport.

Reply:
Well, that's really technically a problem with configuring them via class variables... that might possibly come back to bite us later.  Though I'm finding it hard to think of an alternative.

Perhaps TAP#start and TAP#stop should handle this? Thoughts?